modules/hitl2/src/views/full/Components/Conversation.tsx

```diff
@@ -53,45 +51,31 @@ const Conversation: FC<Props> = props => {
   }
 
   return (
-    <Row className={cx(styles.h100)}>
-      {!props.escalation && (
-        <Col xs={12} className={cx(styles.h100)}>
-          <Tabs
-            tabs={[{ id: 'conversation', title: lang.tr('module.hitl2.tab') }]}
-            style={{ marginBottom: 'var(--spacing-xx-large)' }}
-          />
-          <EmptyState icon={<AgentsIcon />} text={lang.tr('module.hitl2.conversation.empty')}></EmptyState>
-        </Col>
-      )}
+    <Fragment>
+      <div className={style.conversationWrapper}>
+        <Tabs tabs={[{ id: 'conversation', title: lang.tr('module.hitl2.conversation.tab') }]} className={style.tabs} />
+        <div className={style.main}>
+          {!props.escalation ? (
+            <EmptyState icon={<AgentsIcon />} text={lang.tr('module.hitl2.conversation.empty')}></EmptyState>
+          ) : (
+            <Fragment>
+              <Button onClick={handleAssign}>Assign to me</Button>
```

Comment:
I18n

Reply:
Those are just temporary buttons, they will be removed once the chat is added I believe